docs/ldml/tr35.md

```diff
@@ -1368,6 +1368,23 @@ root
 
 _The inheritance is often not simple truncation, as will be seen later in this section._
 
+The default search chain is slighly different for multiple variants. 
+In that case, the inheritance chain covers all combinations of variants, with longest number of variants first. 
+For example, where the requested locale ID is en_fonipa_scouse, the inheritance chain is as follows:
+
+```
+en_GB_fonipa_scouse
+en_GB_scouse_fonipa // extra step, only needed if not canonical
```

Comment:
Question: which is canonical: "en_GB_fonipa_scouse" or "en_GB_scouse_fonipa"? I thought "en_GB_fonipa_scouse" was canonical because the variants are sorted.

Reply:
Yes, en_GB_fonipa_scouse is the canonical one, and checked first. en_GB_scouse_fonipa only needs to be checked if the implementation allows non-canonical locale names in its data (as explained right below). Does that need to be clarified? 

If the data for the implementation performing the inheritance doesn't require canonical locale identifiers, then extra locale IDs need to be inserted in the chain. That is indicated in the example above, marked with "only needed if not canonical".